core/pom.xml

```diff
@@ -50,7 +50,7 @@
               <goal>copy-dependencies</goal>
             </goals>
             <configuration>
-              <includeArtifactIds>org.apache.felix.fileinstall</includeArtifactIds>
+              <includeArtifactIds>org.apache.felix.fileinstall,org.osgi.util.function,org.osgi.util.promise,org.apache.felix.scr</includeArtifactIds>
```

Comment:
The `org.osgi.util.function,org.osgi.util.promise` dependencies are required by the SCR one.